io/plugins/eu.esdihumboldt.hale.io.xls/plugin.xml

```diff
@@ -76,16 +76,19 @@
                ref="eu.esdihumboldt.hale.io.xls.xlsx">
          </contentType>
          <providerParameter
-               description="States if the first line in the CSV file should be skipped (e.g. because it contains the header)"
-               label="Skip first line"
+               description="States how many lines should be skipped when reading instances 
+               from a XLS file (e.g. because of the header). Also boolean values are 
+               supported for backwards compatibility. When a boolean is used, true is translated 
+               into 1 (i.e. skip the first line) and false into 0 (i.e. do not skip any lines)"
+               label="Skip N lines"
                name="skip"
-               optional="true">
+               optional="0">
```

Comment:
I think this should be `true`. Didn't you use the extension editor in Eclipse? I think that would not have let you choose a wrong value.

Reply:
I don't know what the extension editor is, can you please show me? However, in my logic, `skip` is now an integer and that is why I would use default `0`.